client/app/mfaEnrollment/mfaEnrollment.controller.js

```diff
@@ -0,0 +1,18 @@
+export default class mfaEnrollmentCtrl {
+  /* @ngInject */
+  constructor(
+    $state,
+    CORE_MANAGER_URLS,
+    coreConfig,
+  ) {
+    this.$state = $state;
+    const mfaBaseUrl = _.get(CORE_MANAGER_URLS, `${coreConfig.getRegion()}.dedicated`);
+    const mfaUrlPath = '#/useraccount/security/mfa';
```

Comment:
Would be better to `$state.href(<stateName>)` that way any url change won't break the redirection